### PR TITLE
 Display associated tags and custom fields on invoice items #133

### PR DESCRIPTION
**Pull request by wwjbatista**
https://github.com/killbill/killbill-client-ruby/pull/40

### CODE REVIEW

**Comment by sbrossie** [comment]
👍

**Comment by sbrossie** [comment]
👍

File: app/controllers/kaui/invoices_controller.rb [hunk 1]
```diff
@@ -42,20 +42,37 @@ def pagination
   end
 
   def show
-    @invoice = Kaui::Invoice.find_by_id_or_number(params.require(:id), true, 'FULL', options_for_klient)
+    # Go to the database once
```
Comment (pierre) [inline app/controllers/kaui/invoices_controller.rb]: 👍

**Comment by pierre** [comment]
Merged in work-for-release-0.19.x as the new APIs are 0.19-only at the moment.

**Pull request closed without merging**